antismash/modules/rrefinder/test/test_rrefinder.py

```diff
@@ -0,0 +1,297 @@
+        self.hit_a2 = {'locus_tag': 'locus_tag_a', 'location': '[1100:2100]',
+                        'label': 'a1_generic_label', 'domain': 'RRE_type_A', 'evalue': 0.1,
+                        'protein_start': 0, 'protein_end': 40, 'score': 38.0,
+                        'description': 'description_type_A', 'translation': 'FAKESEQ',
+                        'identifier': 'RREFam001',
+                      }
+        self.hit_b1 = {'locus_tag': 'locus_tag_b', 'location': '[500:1500]',
+                        'label': 'b1_generic_label', 'domain': 'RRE_type_B', 'evalue': 0.2,
+                        'protein_start': 0, 'protein_end': 90, 'score': 25.0,
+                        'description': 'description_type_B', 'translation': 'FAKESEQ',
+                        'identifier': 'RREFam002',
+                        }
+        self.hit_b2 = {'locus_tag': 'locus_tag_b', 'location': '[600:1600]',
+                        'label': 'b1_generic_label', 'domain': 'RRE_type_C', 'evalue': 0.2,
+                        'protein_start': 14, 'protein_end': 50, 'score': 42.0,
+                        'description': 'description_type_C', 'translation': 'FAKESEQ',
+                        'identifier': 'RREFam003',
+                      }
+        self.hit_c = {'locus_tag': 'locus_tag_c', 'location': '[200:400]',
+                        'label': 'a_generic_label', 'domain': 'RRE_type_D', 'evalue': 0.2,
+                        'protein_start': 0, 'protein_end': 120, 'score': 6.0,
+                        'description': 'description_type_D', 'translation': 'FAKESEQ',
+                        'identifier': 'RREFam004',
+                      }
+
+        self.candidates_per_protocluster = defaultdict(list)
+        self.candidates_per_protocluster[1] = ['locus_tag_a', 'locus_tag_c']
+        self.candidates_per_protocluster[2] = ['locus_tag_b']
+
+        self.filtered_hits_per_protocluster = defaultdict(list)
+        self.filtered_hits_per_protocluster[1] = ['locus_tag_a']
+        self.filtered_hits_per_protocluster[2] = ['locus_tag_b']
+
+        self.all_hits = [self.hit_a1, self.hit_a2, self.hit_b1, self.hit_b2, self.hit_c]
+        self.fake_cds_info = dict((locus_tag, locus_tag) for locus_tag in ['locus_tag_a', 'locus_tag_b', 'locus_tag_c'])
+        self.hit_info = defaultdict(list)
+        for hit in self.all_hits:
+            self.hit_info[hit['locus_tag']].append(hit)
+        self.filtered_hit_info = dict((hit['locus_tag'], [hit]) for hit in [self.hit_a1, self.hit_b1])
+
+        self.bitscore_cutoff = 25.0
+        self.max_evalue = 1
+        self.min_length = 50
+        self.record_id = 'test_record'
+        self.mock_record = self.make_mock_record()
+        self.database = 'rre_database'
+        self.tool = 'rrefinder'
+        
+        self.hmm_res = HmmerResults(self.record_id, self.max_evalue, self.bitscore_cutoff,
+                                    self.database, self.tool, self.all_hits)
+
+    def make_mock_record(self):
```

Comment:
You could probably skip all of this and use `antismash.common.test.helpers.FakeRecord` and similar.